app/javascript/utils/utils.js

```diff
@@ -37,21 +37,15 @@ export const isChanged = (prev, current) => {
 }
 
 export const filterChanged = (prev, current) => {
-  if (!prev) {
-    return current
-  }
+  if (!prev) return current
   const changedFields = {}
   forEach(current, (value, key) => {
     if (!isEqual(prev[key], value)) {
       // date fields need to be treated as whole
       if (isPlainObject(value) && prev[key] !== null && !isDateObject(value)) {
```

Comment:
non-blocking: another option with the same functionality that would align with previous logic: add `key !== has_ada_priorities_selected` to this if statement